backend/handler/password.go

```diff
@@ -49,6 +56,14 @@ func (h *PasswordHandler) Set(c echo.Context) error {
 		return dto.NewHTTPError(http.StatusBadRequest, "failed to parse userId as uuid").SetInternal(err)
 	}
 
+	pwBytes := []byte(body.Password)
+	if utf8.RuneCountInString(body.Password) < h.cfg.MinPasswordLength { // use utf8.RuneCountInString, so utf8 characters would count as 1
+		return dto.NewHTTPError(http.StatusBadRequest, fmt.Sprintf("password must be at least %d characters long", h.cfg.MinPasswordLength))
+	}
+	if len(pwBytes) > 72 {
```

Comment:
https://cs.opensource.google/go/x/crypto/+/master:bcrypt/bcrypt_test.go;l=67
The actual max Length of this implementation seems to be 55

Reply:
Hmm interressting. I wrote some example test and it works with 72 bytes.

```
func TestName(t *testing.T) {
	hash, err := bcrypt.GenerateFromPassword([]byte("012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890"), 12)
	assert.NoError(t, err)
	println(string(hash))

	err = bcrypt.CompareHashAndPassword(hash, []byte("012345678901234567890123456789012345678901234567890123456789012345678901"))
	assert.NoError(t, err)
}
```

So the first password has more than 72 bytes and it cuts off the everything greater than 72 bytes, because the second password has exactly 72 bytes. If you remove the last byte/character (the "1") then the test fails. So I think the limit is 72 bytes. See also https://github.com/golang/go/issues/36546

Reply:
Alright! Could you add that test to the suite for documentation purposes? 👍